PeerioTechnologies/peerio-desktop
# WIP: Futurio signing

Add signing with Futurio-only secret key stored in peerio-desktop-futurio repo, encrypted with password.

**IMPORTANT: updater needs normal (release, staging, etc.) testing**

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  

https://peerio.tpondemand.com/entity/7524

#### Linked PRs in other repos

https://github.com/PeerioTechnologies/peerio-desktop-futurio/pull/2
 
#### Manual testing instructions  

* Check that nightly releases work.
* Check that nightly releases self-update.
* **Check that normal updates (staging, release) work, especially on Mac.**


----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
